site/en/blog/new-in-chrome-120/index.md

```diff
@@ -0,0 +1,102 @@
+
+Web developers had no good way to handle close requests for their own components. This is especially problematic on Android devices, where providing the simple closing behavior for the back gesture is quite complex.
+
+Chrome 120 brings the solution with CloseWatcher, a new API for directly listening and responding to close requests. It also includes upgrades to [`<dialog>`](https://developer.mozilla.org/docs/Web/HTML/Element/dialog)  and [`popover=""`](https://developer.mozilla.org/docs/Web/API/Popover_API) to use the new close request framework, so that they respond to the Android back button.
+
+Checkout [the CloseWatcher API demo](https://close-watcher-demo.glitch.me/) to give it a try.
+
+## `<details>` name attribute {: #details-name }
+
+The `name` attribute for the [`<details>`](https://developer.mozilla.org/docs/Web/HTML/Element/details) element makes it easy to implement the accordion pattern using a sequence of `<details>` HTML elements.
+
+Multiple `<details>` elements that have the same `name` form a group. With this setup at most one element in that group can be opened at once.
+
+Here is an example with a group that shares the name `cookies`:
+
+```html
+<details>
+  <summary>Chocolate chip</summary>
+  Yum yum chocolate chip.
+</details>
+<details>
+  <summary>Snickerdoodle</summary>
+   Yum yum snickerdoodle.
+</details>
+<details>
+  <summary>Maicenitas</summary>
+   Yum yum maicenitas.
+</details>
+<details>
+  <summary>Sugar cookies</summary>
+   Yum yum sugar cookies.
+</details>
+
+```
+
+## Permission policy violation reports {: #policy-violation-reports }
+
+Permissions policy violation reports are now available, these reports integrate the [Permissions policy API](https://developer.mozilla.org/docs/Web/HTTP/Permissions_Policy) that allows developers to control the browser features available to a page, its iframes, and subresources, by declaring a set of policies for the browser to enforce with the [Reporting API](https://developer.mozilla.org/docs/Web/API/Reporting_API). The Reporting API provides a generic reporting mechanism for web applications to use to make reports available based on several platform features.
+
+This Permissions Policy API and Reporting API integration allows web developers to configure endpoints, to which permissions policy violation reports will be sent, allowing site owners to see when disallowed features are being requested for their pages in the field.
+
+[Controlling browser features with Permissions Policy](/articles/permissions-policy/) includes more implementation details.
+
+
+## And more! {: #more }
+
+Of course there’s plenty more.
+
+* The [relaxed CSS nesting implementation](/blog/css-nesting-relaxed-syntax-update/) allows nested style rules to begin with an element, rather than being wrapped with is() or requiring an ampersand in front.
```

Comment:
```suggestion
* The [relaxed CSS nesting implementation](/blog/css-nesting-relaxed-syntax-update/) allows nested style rules to begin with an element, rather than being wrapped with `is()` or requiring an ampersand in front.
```